api/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/api/kvittering/KvitteringRoute.kt

```diff
@@ -122,10 +122,10 @@ fun RouteExtra.kvitteringRoute() {
     }
 }
 
-private fun tilKvitteringResponse(innsendtInntektsmelding: InnsendtInntektsmelding): KvitteringResponse =
-    KvitteringResponse(
+private fun tilKvittering(innsendtInntektsmelding: InnsendtInntektsmelding): Kvittering =
+    Kvittering(
         kvitteringDokument = innsendtInntektsmelding.dokument?.let { inntektsmeldingDokument ->
-            KvitteringDokument(
+            KvitteringSimba(
```

Comment:
Ingen biggie, men KvitteringSimba er kanskje litt obskurt klassenavn?

Reply:
Jo, men jeg fant ikke noe bedre som tilsa at det var systemet Simba sin versjon av kvittering. Spinn har vel sin egen, og man kunne forventet at det kom fra Storebror også 🤷 Åpen for forslag!

Reply:
kommer ikke på noe bedre enn KvitteringIntern og det er litt meh :/